### DISCUSSION TITLE
[ERROR] Could not resolve "#apg-lite"

### DISCUSSION THREAD

**Discussion by edsonmartins** [swagger-api/swagger-ui#9981] [General]
We updated our React application to the latest version of the component and now we are giving the following error:

[ERROR] Could not resolve "#apg-lite"

    node_modules/.pnpm/openapi-path-templating@1.5.1/node_modules/openapi-path-templating/es/parse/callbacks/path-template.mjs:1:39:
      1 │ import { identifiers, utilities } from '#apg-lite';
        ╵                                        ~~~~~~~~~~~

  The remapped path "apg-lite" could not be resolved:

    node_modules/.pnpm/openapi-path-templating@1.5.1/node_modules/openapi-path-templating/package.json:18:16:
      18 │       "import": "apg-lite",
         ╵                 ~~~~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "apg-lite" here because it's not listed as a
  dependency of this package:

    ../../../../.pnp.cjs:36:31:
      36 │         "packageDependencies": [\
         ╵                                ~~

  You can mark the path "#apg-lite" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "#apg-lite"

    node_modules/.pnpm/openapi-path-templating@1.5.1/node_modules/openapi-path-templating/es/parse/callbacks/path-literal.mjs:1:39:
      1 │ import { identifiers, utilities } from '#apg-lite';
        ╵                                        ~~~~~~~~~~~

  The remapped path "apg-lite" could not be resolved:

    node_modules/.pnpm/openapi-path-templating@1.5.1/node_modules/openapi-path-templating/package.json:18:16:
      18 │       "import": "apg-lite",
         ╵                 ~~~~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "apg-lite" here because it's not listed as a
  dependency of this package:

    ../../../../.pnp.cjs:36:31:
      36 │         "packageDependencies": [\
         ╵                                ~~

  You can mark the path "#apg-lite" as external to exclude it from the bundle, which will remove
  this error.

**Comment by regisa** [comment]
FYI, I get this error when using turbopack, works fine with webpack (nextjs, using `next dev` vs `next dev --turbo`)
